integreat_cms/cms/templates/imprint/imprint_sbs.html

```diff
@@ -160,10 +171,12 @@ <h1 class="heading">
             </div>
         </div>
     </form>
+    {{ media_config_data|json_script:"media_config_data" }}
     {% if not perms.cms.change_imprintpage %}
         {% include "../_tinymce_config.html" with readonly=1 language=target_language %}
     {% else %}
         {% include "../_tinymce_config.html" with language=target_language %}
+        {% include "../_content_edit_lock.html" with lock_key=source_imprint_translation.page.edit_lock_key %}
     {% endif %}
```

Comment:
```suggestion
    {% endif %}
    {% include "../generic_confirmation_dialog.html" %}
```
The confirmation dialog also needs to be included